source/extensions/filters/listener/tls_inspector/tls_inspector.h

```diff
@@ -62,6 +69,7 @@ class Config {
 private:
   TlsInspectorStats stats_;
   bssl::UniquePtr<SSL_CTX> ssl_ctx_;
+  const std::unordered_set<uint32_t> inspected_ports_;
```

Comment:
I'm pretty sure that using an `unordered_set` is an overkill here, given that the list of ports should be less than a 100. Arrays are superior to maps when N is small (because they fit in the CPU cache).

I recommend using either [`absl::FixedArray`](https://github.com/abseil/abseil-cpp/blob/master/absl/container/fixed_array.h) or` std::vector` instead.